netjsonconfig/backends/base/converter.py

```diff
@@ -109,6 +109,19 @@ def to_netjson(self, remove_block=True):
         # return result, expects dict
         return result
 
+    def clean_intermediate_data(self, intermediate_data):
```

Comment:
you're adding a fix for the OpenWrt backend in the Base converter.

Base classes are supposed to be generic.

I am going to handle this issue on my own. Thanks for the fix.

Reply:
That was actually a conscious decision, I thought other backends like (openwisp 1.X) will need this change as well, however, I have not worked with any other backend, so I might have misunderstood it incorrectly.
Thanks for the information! :smile:

Reply:
OpenWrt derivatives will have to install OpenWrt base classes.

Don't forget it's possible to implement backends for non-openwrt systems.
We implemented two experimental backends in the past:

- https://github.com/openwisp/netjsonconfig/tree/raspbian
- https://github.com/openwisp/netjsonconfig/tree/airos

The goal of OpenWISP is to support multiple systems.